be/src/exec/vectorized/connector_scan_node.cpp

```diff
@@ -134,7 +134,13 @@ pipeline::OpFactories ConnectorScanNode::decompose_to_pipeline(pipeline::Pipelin
     auto&& rc_rf_probe_collector = std::make_shared<RcRfProbeCollector>(1, std::move(this->runtime_filter_collector()));
     this->init_runtime_filter_for_operator(scan_op.get(), context, rc_rf_probe_collector);
 
-    return pipeline::decompose_scan_node_to_pipeline(scan_op, this, context);
+    auto operators = pipeline::decompose_scan_node_to_pipeline(scan_op, this, context);
+
+    if (!_data_source_provider->insert_local_exchange_operator()) {
```

Comment:
@silverbullet233  Is that what you expect? Because JDBCScanNode does not have any scan ranges, you want to inserrt a local exchange operator to increase parallelism, right?

Reply:
> @silverbullet233 Is that what you expect? Because JDBCScanNode does not have any scan ranges, you want to inserrt a local exchange operator to increase parallelism, right?

yes. JDBCScanNode doesn't have scan ranges and its dop is 1. @dirtysalt